src/ResourceManager/Network/Commands.Network/Common/NFVBaseCmdlet.cs

```diff
@@ -0,0 +1,48 @@
+﻿
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using AutoMapper;
+using Microsoft.Azure.Commands.Network.Models;
+using Microsoft.Azure.Commands.Tags.Model;
+using Microsoft.Azure.Management.Network;
+using System.Net;
+using System.Management.Automation;
+using System;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    using Microsoft.Azure.Commands.Network.Common;
+
+    public abstract class NFVBaseCmdlet : NetworkBaseCmdlet
+    {
+        public override void ExecuteCmdlet()
+        {
+            base.ExecuteCmdlet();
```

Comment:
Put this inside NetworkBaseCmdlet. We dont need NFVBaseCmdlet. Yes, this would impact all Network cmdlets but this is a good change